### PR TITLE
few fixes and updates

### PR DESCRIPTION
**Pull request by tjavaid**
Few updates:

- datacards config files updated with LLR rates
- a fix in [producePlots.py](https://github.com/ram1123/Fiducial_XS/blob/debugStatUncIssue/python/producePlots.py) , [plotLHScans.py](https://github.com/ram1123/Fiducial_XS/blob/debugStatUncIssue/python/plotLHScans.py) and [runHZZFiducialXS.py](https://github.com/ram1123/Fiducial_XS/blob/debugStatUncIssue/runHZZFiducialXS.py) macros.

### CODE REVIEW

File: python/plotLHScans.py [hunk 1]
```diff
@@ -487,4 +488,4 @@ def parseOptions():
                 f.write('resultsXS = '+str(resultsXS)+' \n')
             with open(datacardInputs+'/resultsXS_LHScan_observed_'+obsName.replace(' ','_')+'_v3.py', 'w') as f:
                 f.write('resultsXS = '+str(resultsXS_data)+' \n')
-    binCounter += 1
+        binCounter += 1
```
Comment (vukasinmilosevic) [inline python/plotLHScans.py:491]: was this a bug?

File: runHZZFiducialXS.py [hunk 2]
```diff
@@ -1396,7 +1397,8 @@ def runFiducialXS():
...
 
-                cmd = "combine -n "+obsName.replace(' ','_')+"_SigmaBin"+str(obsBin)+"_NoSys"+'_'+str(year)+unblindString+" -M MultiDimFit -d "+combineOutputs.format(year = year)+"/SM_125_all_13TeV_xs_"+obsName.replace(' ','_')+"_bin_"+physicalModel+unblindString+"_result.root -w w --snapshotName \"MultiDimFit\" -m 125.38 -D toy_asimov --setParameters MH=125.38 -P SigmaBin"+str(obsBin)+" --floatOtherPOIs=1 --saveWorkspace --setParameterRanges MH=125.38,125.38:SigmaBin"+str(obsBin)+"=0.0,3.0 --redefineSignalPOI SigmaBin"+str(obsBin)+" --algo=grid --points=50 --autoRange 4 "
+                #cmd = "combine -n "+obsName.replace(' ','_')+"_SigmaBin"+str(obsBin)+"_NoSys"+'_'+str(year)+unblindString+" -M MultiDimFit -d "+combineOutputs.format(year = year)+"/SM_125_all_13TeV_xs_"+obsName.replace(' ','_')+"_bin_"+physicalModel+unblindString+"_result.root -w w --snapshotName \"MultiDimFit\" -m 125.38 -D toy_asimov --setParameters MH=125.38 -P SigmaBin"+str(obsBin)+" --floatOtherPOIs=1 --saveWorkspace --setParameterRanges MH=125.38,125.38:SigmaBin"+str(obsBin)+"=0.0,3.0 --redefineSignalPOI SigmaBin"+str(obsBin)+" --algo=grid --points=50 --autoRange 4 "
```
Comment (vukasinmilosevic) [inline runHZZFiducialXS.py:1418]: Did I understand correctly, it used asimov toy dataset for the uncertainty computation before?

**Review by vukasinmilosevic — CHANGES_REQUESTED** [review]

File: python/plotLHScans.py (hunk 1 above)
Comment (tjavaid) [reply python/plotLHScans.py:491]: yes, somehow it was out of loop and only printed the first bin's width on LScan plots for all bins.

File: runHZZFiducialXS.py (hunk 2 above)
Comment (tjavaid) [reply runHZZFiducialXS.py:1418]: the unblinding central value was coming from the `toy_Asimov` and in the likelihood scan plot, the blinded and unblinded scans were overlapping each other.

**Review by vukasinmilosevic — APPROVED** [review]

**Pull request merged**